api/config.js

```diff
@@ -15,7 +15,9 @@ var config = {
     redis: {
       host: process.env.REDIS_HOST || 'localhost',
       port: process.env.REDIS_PORT || 6379,
+      password: process.env.REDIS_PASSWORD,
       userSessionTrackingEnabled: process.env.ENABLE_USER_SESSION_TRACKING || false,
+      tls: process.env.REDIS_TLS ? { port: process.env.TLS_PORT || 6380 } : undefined,
```

Comment:
Looks like the docs have `REDIS_TLS_PORT`(which makes more sense) but you use `TLS_PORT` here